core/src/main/kotlin/behavior/GuildInteractionBehavior.kt

```diff
@@ -11,8 +11,4 @@ interface GuildInteractionBehavior : InteractionBehavior {
 
     suspend fun getGuild(): Guild = supplier.getGuild(guildId)
 
-    companion object {
-
-    }
-
```

Comment:
Why'd we end up deleting this? Companion objects are useful for third-party extension functions: see [Companion object for Boolean](https://youtrack.jetbrains.com/issue/KT-7922).